multi_import/formats.py

```diff
@@ -156,3 +156,10 @@ def write(self, dataset):
     csv,
     txt,
 )
+
+supported_mimetypes = (
+    'text/plain',
+    'text/csv',
+    'application/vnd.ms-excel',
+    'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet',
```

Comment:
Does this include Apple Pages documents?

Reply:
I don't _think_ so?

According to https://stackoverflow.com/questions/1454777/apple-iwork-mime-types

I got this list from https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types and just took plain text, csv, xls and xlsx from it.

Reply:
I guess we don't claim we support Pages anyway, just .csv and .xlsx so this is probably fine. I'll start dev qa